packages/runtime/client/rest/src/contract/rolesConfigurations.ts

```diff
@@ -0,0 +1,240 @@
+import * as t from "io-ts/lib/index.js";
+
+import { PolicyId, RoleName } from "@marlowe.io/language-core-v1";
+
+import * as G from "@marlowe.io/language-core-v1/guards";
+
+/**
+ *  @category Roles Configuration
+ */
+export interface AddressBech32Brand {
+  readonly AddressBech32: unique symbol;
+}
+
+export const AddressBech32Guard = t.brand(
+  t.string,
+  (s): s is t.Branded<string, AddressBech32Brand> => true,
+  "AddressBech32"
+);
+
+/**
+ *  Cardano Address in a Bech32 format
+ *  @category Roles Configuration
+ */
+export type AddressBech32 = t.TypeOf<typeof AddressBech32Guard>;
+
+/**
+ *  Definition a of Closed Role Tlken
+ *  @remarks
+ *    - It is only defined by the Address where the token is distributed
+ *  @category Roles Configuration
+ */
+export type ClosedRole = AddressBech32;
+export const ClosedRoleGuard: t.Type<ClosedRole, string> = AddressBech32Guard;
+
+/**
+ *  Definition of an Open Role Token
+ *  @category Roles Configuration
+ */
+export type OpenRole = "OpenRole";
+export const OpenRoleGuard = t.literal("OpenRole");
+
+/**
+ *  Construction of an Open Role Token
+ *  @category Roles Configuration
+ */
+export const openRole = "OpenRole";
+
+/**
+ *  Definition of the Openness of a Role Token
+ *  @category Roles Configuration
+ */
+export type Openness = ClosedRole | OpenRole;
+export const OpenessGuard: t.Type<Openness, string> = t.union([
```

Comment:
guards needs category to render properly

Reply:
https://github.com/input-output-hk/marlowe-ts-sdk/issues/118